packages/graph/hash_graph/bench/benches/representative_read/mod.rs

```diff
@@ -57,6 +60,106 @@ fn bench_representative_read_entity(c: &mut Criterion) {
 }
 
 #[criterion]
+fn bench_representative_read_multiple_entities(c: &mut Criterion) {
+    let mut group = c.benchmark_group("representative_read_multiple_entities");
+    let (runtime, store_wrapper) = setup(DB_NAME, false, false);
+    group.sample_size(10);
+    group.sampling_mode(SamplingMode::Flat);
+    // Set the warm_up and measurement times to try and encourage Criterion to run as few iterations
+    // as possible as these benchmarks run a *lot* longer than the intended Criterion benchmark,
+    // by a few orders of magnitude
+    group.warm_up_time(Duration::from_nanos(1));
+    group.measurement_time(Duration::from_millis(100));
+
+    let graph_resolve_depths = [
+        GraphResolveDepths {
+            data_type_resolve_depth: 0,
+            property_type_resolve_depth: 0,
+            entity_type_resolve_depth: 0,
+            entity_resolve_depth: 0,
+        },
+        GraphResolveDepths {
+            data_type_resolve_depth: 0,
+            property_type_resolve_depth: 0,
+            entity_type_resolve_depth: 2,
+            entity_resolve_depth: 0,
+        },
+        GraphResolveDepths {
+            data_type_resolve_depth: 0,
+            property_type_resolve_depth: 2,
+            entity_type_resolve_depth: 2,
+            entity_resolve_depth: 0,
+        },
+        GraphResolveDepths {
+            data_type_resolve_depth: 2,
+            property_type_resolve_depth: 2,
+            entity_type_resolve_depth: 2,
+            entity_resolve_depth: 0,
+        },
+        GraphResolveDepths {
+            data_type_resolve_depth: 2,
+            property_type_resolve_depth: 2,
+            entity_type_resolve_depth: 2,
+            entity_resolve_depth: 2,
+        },
+        GraphResolveDepths {
+            data_type_resolve_depth: 255,
+            property_type_resolve_depth: 255,
+            entity_type_resolve_depth: 255,
+            entity_resolve_depth: 255,
+        },
+    ];
+
+    for graph_resolve_depth in graph_resolve_depths {
+        group.bench_with_input(
+            BenchmarkId::new(
+                "entity_by_property",
+                format!(
```

Comment:
Did you try just serializing it here? Might be nicer if it's machine parseable

Reply:
I made this shorter than the serialized value as they got trimmed in the report. I'm happy to serialize it if you like